recap/hms/hms.py

```diff
@@ -0,0 +1,39 @@
+from recap.hms.hive_metastore import ThriftHiveMetastore
+from hive_metastore.ttypes import *
+from thrift.transport import TTransport, TSocket
+from thrift.protocol.TBinaryProtocol import TBinaryProtocol
+
+class _HMS:
```

Comment:
Was this autogenerated?

Reply:
The _HMS no, I added it as a way to reduce the surface area of the thrift client that is exposed through Recap. The client has a lot of functionality that is not necessarily needed, e.g. to create a catalog or to operate on partitions etc.